File: samples/sample.py
// Sample JavaScript code for testing conversion
def calculateSum(numbers):
    sum = 0
    for (i = 0; i < numbers.__len__(); i += 1) :        sum += numbers[i]

    print(f"The sum is: ${sum}")
    return sum

    numbers = [1, 2, 3, 4, 5]
    result = calculateSum(numbers)
    print(f"Result: ${result}")

File: samples/sample.js
// Sample JavaScript code for testing conversion
function calculateSum(numbers) {
    let sum = 0;
    for (let i = 0; i < numbers.length; i++) {
        sum += numbers[i];
    }
    console.log(`The sum is: ${sum}`);
    return sum;
}

const numbers = [1, 2, 3, 4, 5];
const result = calculateSum(numbers);
console.log(`Result: ${result}`);